docker/test/integration/minifi/core/TcpClientContainer.py

```diff
@@ -0,0 +1,23 @@
+import logging
+from .Container import Container
+
+
+class TcpClientContainer(Container):
+    def __init__(self, name, vols, network, image_store, command=None):
+        super().__init__(name, 'tcp-client', vols, network, image_store, command)
```

Comment:
We actually do this in `ListenTcpTests.cpp` where we test the processor's functionality. I think it still has some added value as in the docker test we have a system level test instead of the unit level like in ListenTcpTests where we only test the processor separately. This way we can be sure that the processor is registered and works on a system level when defined in a flow configuration and can listen to messages on a separate network dedicated to it in the docker environment.